java/kotlin-extractor2/src/main/kotlin/entities/Function.kt

```diff
@@ -100,98 +143,105 @@ fun KotlinUsesExtractor.getFunctionLabel(
             // default to this function's value parameters' types.
             // (Return-type wildcard addition is always off by default)
             addParameterWildcardsByDefault: Boolean,
-            // The prefix used in the label. "callable", unless a property label is created, then it's
-            // "property".
-            prefix: String = "callable"
     */
+    // The prefix used in the label. "callable", unless a property label is created, then it's
+    // "property".
+    prefix: String = "callable"
 ): String {
-    /*
-    OLD: KE1
-            val allParamTypes =
-                if (extensionParamType == null) parameterTypes
-                else listOf(extensionParamType) + parameterTypes
-
-            val substitutionMap =
-                classTypeArgsIncludingOuterClasses?.let { notNullArgs ->
-                    if (notNullArgs.isEmpty()) {
-                        null
-                    } else {
-                        val enclosingClass = getEnclosingClass(parent)
-                        enclosingClass?.let { notNullClass ->
-                            makeTypeGenericSubstitutionMap(notNullClass, notNullArgs)
-                        }
-                    }
+
+    val allParamTypes = (extensionParamType?.let { listOf(it) } ?: listOf()) + parameterTypes
```

Comment:
I think this can be written more simply as `listOfNotNull(extensionParamType) + parameterTypes`